hack/util-install-bundle.sh

```diff
@@ -18,6 +18,19 @@ metadata:
   name: build-pipeline-selector
 spec:
   selectors:
+    - name: Hermetic build (docker)
+      pipelineParams:
+        - name: hermetic
+          value: "true"
+        - name: prefetch-input
+          value: "gomod"
+      pipelineRef:
+        name: docker-build
+        bundle: ${BUNDLE}
+      when:
+        dockerfile: true
+        language: Go
+        projectType: test-hermetic-build
```

Comment:
For my understanding, does this mean we want to test hermetic builds only for Go? (for now?)

Reply:
yes, that was my understanding. I think it shouldn't be a problem to add a python repo too though (not sure if it should be done as part of this one, or a follow-up ticket).
@Michkov thoughts?

Reply:
I filed a separate one for pip: STONEBLD-694